tl/features/create_pseudo_gt.py

```diff
@@ -1,17 +1,24 @@
 import pandas as pd
 from tl.exceptions import RequiredColumnMissingException
+from tl.file_formats_validator import FFV
+from tl.exceptions import UnsupportTypeError
 
 
-def create_pseudo_gt(df: pd.DataFrame, column_thresholds: list,
+def create_pseudo_gt(df: pd.DataFrame, column_thresholds: str,
                      output_column: str):
+    column_thresholds = [(_.split(":")[0], float(_.split(":")[1]))
```

Comment:
one last change, I see what you are trying to do, but you are splitting `_` twice where you can split it once and access both values in one go. A for loop will help here